doc/installation/index.rst

```diff
@@ -4,8 +4,8 @@
 Installation
 ############
 
-Centreon recommends using its official packages from the Centreon
-Entreprise Server (CES) repository. Most of Centreon' endorsed
+Centreon recommends using its official packages from the Centreon Open Sources
+version available free of charge on our repository (ex CES). Most of Centreon endorsed
```

Comment:
COS instead of CES ??

Reply:
I will remove "(ex CES)", is based in another documentantion in Centreon-web. Thanks